iree/schemas/dylib_executable_def.fbs

```diff
@@ -13,6 +13,11 @@
 // limitations under the License.
 
 namespace iree;
+// Define kinds of sanitizer used to connect CPU kernels and runtime
+enum Sanitizer : ubyte {
+  kNone = 0,
```

Comment:
the naming convention in flatbuffers is no k here (I know it's confusing - every language wants its own style and there's like 4 in this project :) - so just `None`/`Address`.